src/promotion-builder/common/banner/Banner.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropsTypes from 'prop-types';
+
+import './banner.css';
+import block from 'bem-cn';
+const b = block('banner');
+
+const Banner = ({subHeading}) =>  (<header className={b()}><h1>Promotion Manager</h1><h3>{subHeading}</h3></header>);
```

Comment:
I would have this on separate lines to promote legibility. This will have an added benefit making the code look simpler.